oak_remote_attestation_verification/src/tests.rs

```diff
@@ -17,40 +17,80 @@
+};
+
+const TEST_ATTESTATION_ENDORSEMENT: AttestationEndorsement = AttestationEndorsement {
+    tee_certificates: vec![],
+    binary_attestation: None,
+    application_data: None,
+};
+const TEST_REFERENCE_VALUE: ReferenceValue = ReferenceValue {
+    binary_hash: vec![],
+};
+const BINARY_HASH: &str = "39051983bbb600bbfb91bd22ee4c976420f8f0c6a895fd083dcb0d153ddd5fd6";
+
+struct TestData {
+    endorsement_bytes: Vec<u8>,
+    log_entry_bytes: Vec<u8>,
+    rekor_key_pem_bytes: Vec<u8>,
+    endorser_key_pem_bytes: Vec<u8>,
+}
+
+fn load_testdata() -> TestData {
     let endorsement_path = "testdata/endorsement.json";
 
     // LogEntry downloaded from
-    // https://rekor.sigstore.dev/api/v1/log/entries/bb05be1bd813f8afb7b77b2d9f7be5ae25b396d111c7a26a04b785c48c277372.
-    // This LogEntry was created by sining the example endorsement file above,
-    // and publishing that to Rekor (https://rekor.sigstore.dev), as described
+    // https://rekor.sigstore.dev/api/v1/log/entries/24296fb24b8ad77a51d549703a3a1c2dd2639ba49617fc563854031cb93e6d354e7b005065c334a8.
+    // This LogEntry was created by signing the example endorsement file above, using google-cloud
+    // KMS, and uploading it to Rekor (https://rekor.sigstore.dev) using `rekor-cli`, as described
     // in https://github.com/sigstore/rekor/blob/main/types.md#pkixx509.
     // The resulting entry in Rekor has UUID
-    // `bb05be1bd813f8afb7b77b2d9f7be5ae25b396d111c7a26a04b785c48c277372` and logIndex 1323526.
-    // The body of LogEntry can be fetched using `rekor-cli get --log-index 1323526`.
+    // `24296fb24b8ad77a51d549703a3a1c2dd2639ba49617fc563854031cb93e6d354e7b005065c334a8` and
+    // logIndex 30891523. The body of LogEntry can be fetched using `rekor-cli get --log-index
+    // 30891523`.
     let log_entry_path = "testdata/logentry.json";
 
-    // Test public key, representing the product team that signed the endorsement file.
-    let pubkey_path = "testdata/ec_public.pem";
+    // Public key fetched from Google Cloud KMS, associated with the signing key that was used for
+    // signing the endorsement statement.
+    let pubkey_path = "testdata/oak-development.pem";
 
-    // Public key of the Rekor instance hosted by sigstore.dev. It is downloaded from https://rekor.sigstore.dev/api/v1/log/publicKey.
+    // Public key of the Rekor instance hosted by sigstore.dev. It is downloaded from
+    // https://rekor.sigstore.dev/api/v1/log/publicKey.
     let rekor_pubkey_path = "testdata/rekor_public_key.pem";
 
     let endorsement_bytes = fs::read(endorsement_path).expect("couldn't read endorsement file");
     let log_entry_bytes = fs::read(log_entry_path).expect("couldn't read log entry file");
-    let rekor_pem_bytes =
+    let rekor_key_pem_bytes =
         fs::read(rekor_pubkey_path).expect("couldn't read Rekor's public key file");
-    let pubkey_pem_bytes =
+    let endorser_key_pem_bytes =
```

Comment:
I think these variables can also have `_public_` in the name.

Reply:
Thanks. Done.